src/libsodium/crypto_vrf/ietfdraft12/vrf_ietfdraft12.h

```diff
@@ -0,0 +1,10 @@
+#ifndef vrf_ietfdraft12_H
+#define vrf_ietfdraft12_H
+
+static const unsigned char SUITE = 0x04; /* ECVRF-ED25519-SHA512-ELL2 */
+
+static const unsigned char ZERO = 0x00;
```

Comment:
Just use 0, 2 and 3 :)

Reply:
Nevermind. Them being uppercase made me think they were macros, and not variables you need to use the address of.